source/upgrade/important-upgrade-notes.rst

```diff
@@ -2,13 +2,29 @@ Important Upgrade Notes
 
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v6.0                                               | Longer migration times can be expected. See `this document <https://gist.github.com/streamer45/59b3582118913d4fc5e8ff81ea78b055>`_ for more details.             |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Please see `the changelog <https://docs.mattermost.com/install/self-managed-changelog.html>`_ for a list deprecations in this release.                           |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Changed the field type of Data in ``model.ClusterMessage`` to []byte from string. This also makes a v6 server incompatible with a v5 server in the same cluster. |
+|                                                    | Customers upgrading from 5.x to 6.x cannot do an HA upgrade. This will require a downtime.                                                                       |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Focalboard v0.9.1 (released with Mattermost v6.0) requires Mattermost v5.38+.                                                                                    |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Advanced logging configuration schema changed. This is a breaking change relative to 5.x. See updated documentation.                                             |
```

Comment:
@cwarnermm, this release note references updated documentation for logging config. Did that get done?  See here: https://community.mattermost.com/core/pl/ufhjhrm36irx9qzi7fpi4w91oc

Reply:
Thanks, @wiggin77! @wiersgallak is developing this doc update for v6.0.